core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/resolvers/ExternalLinkResolverImpl.java

```diff
@@ -79,6 +73,11 @@ public class ExternalLinkResolverImpl implements LinkResolver {
                 description = "Example - Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like " +
                         "Gecko) Chrome/86.0.4240.111 Safari/537.36"
         ) String userAgent() default StringUtils.EMPTY;
+
+        @AttributeDefinition(
+                name = "Activate service",
```

Comment:
(nerdy) Use just the word `"Enable"`. Service is _always_ active unless it is stopped like in Felix console. "Activate" is a technical term in OSGi. This here is not about "activation". A service IS active but it CAN be not enabled ==> can be returning no links

Reply:
done